roadmap.md

```diff
@@ -1,24 +1,38 @@
 # Tekton Triggers Roadmap
 
-In 2019 we created a simple system for creating instances of Tekton resources, triggered
-by json payloads sent to HTTP endpoints (Tekton Triggers).
+In 2021, we'd like to get the Triggers APIs to a beta level of stability, add popular 
+feature requests, and plan for a v1 release.
 
-In 2020 we would like to add missing features and then push for a `beta` release in the
-same year.
+## H1 2021:
+* Get to Beta!
+    * [Trigger Templates - JSON escaping, validation, and templating](https://github.com/tektoncd/triggers/issues/697) 
+    * Trigger CRD - Path based Triggers
+    * v1beta1 version upgrade
+    * Beta docs/examples overhaul
 
-We are targeting improving the experience for both end users and operators:
+* Operator features:
+    * RBAC/Permission setup for multi-tenant usage
```

Comment:
Is there an issue describing the desired state here? I understand the goal, but would love to see something that outlines what the general RBAC principals would look like here.

Reply:
https://github.com/tektoncd/triggers/issues/77 is the issue